### ISSUE TITLE
Azul chrashes after installation on MacOS 10.13

### ISSUE THREAD

**Issue by GrohmannDaniel** [tudelft3d/azul#10]
I installed AZUL from Apple AppStore today. When I start AZUL it crashes immediately. A program window  doesn't appear. I attached the error report to my message. 

Did you know what's going wrong with AZUL's installation on my Mac?

regards
Daniel
[errorreport_azul.txt](https://github.com/tudelft3d/azul/files/3303066/errorreport_azul.txt)

**Comment by kenohori** [comment]
Sorry, but I'm honestly not sure about what's going on. There seems to be an issue with how notifications are handled, but there's nothing obvious in the log.

I'll try to look into it anyway.

**Comment by ImkeLansky** [comment]
Hi,

I'm having a similar problem on MacOS 10.14.6. I'm on the latest version of Azul, v0.9. The previous version, v0.8.1, doesn't crash. As soon as I open v0.9 and click on the 'display window' of the app, or when I'm trying to open a file, the app crashes. The app is installed through the App Store. I attached two crash report messages, one that was generated when opening a file and the other for clicking on the app window.

Kind regards,

Imke
[crash_report_open_file.txt](https://github.com/tudelft3d/azul/files/3520724/crash_report_open_file.txt)
[crash_report_startup.txt](https://github.com/tudelft3d/azul/files/3520725/crash_report_startup.txt)

**Comment by hugoledoux** [comment]
and if you download the code and compile it yourself? It's trivial, all dependencies are there and there's an XCode project.

The only thing to change is the developers ID, the compiler will complain about this and then just put your Apple developers ID (you do need one, it's free).

![2019-08-21 at 10 30](https://user-images.githubusercontent.com/1546518/63415896-b9555e80-c3fe-11e9-8ae4-a00d3b4c7c8e.jpg)

**Comment by ImkeLansky** [comment]
Thank you Hugo. I tried the approach, but my XCode is having problems with the libraries. I could get boost and cgal to work, however, pugixml keeps giving me errors even though it is installed and in the same folder as boost and cgal. All paths are included, but it still cannot find the .hpp file.

**Comment by hugoledoux** [comment]
hmmm, but all dependencies are in `/libs/` so you shouldn't have to install anything.

Ken is on holidays, so if problems you'll have to wait a 1w I'm afraid, I just downloaded the project and it compiled and ran without problems.

**Comment by ImkeLansky** [comment]
Yes I thought so as well, but it seems to ignore the `/libs/` folder for some reason. It is included in the library search paths in the project settings. And I also see them in the 'Linked Frameworks and Libraries' option in the settings.
